pkg/services/ngalert/schedule/schedule_unit_test.go

```diff
@@ -38,9 +38,9 @@ func TestSchedule_ruleRoutine(t *testing.T) {
 	createSchedule := func(
 		evalAppliedChan chan time.Time,
 		senderMock *AlertsSenderMock,
-	) (*schedule, *store.FakeRuleStore, *store.FakeInstanceStore, prometheus.Gatherer) {
+	) (*schedule, *store.FakeRuleStore, *state.FakeInstanceStore, prometheus.Gatherer) {
```

Comment:
Schedule is already highly coupled to state so I felt it was fine to keep the `state` references and not re-implement the fake.